runtime/data/test_data_with_header_and_footer.csv

```diff
@@ -0,0 +1,11 @@
+header
```

Comment:
You won't have headers like that, in csv in 99% cases num of header cols will match data num cols.